src/App.vue

```diff
@@ -55,10 +56,19 @@ export default defineComponent({
       });
     emitter.on('presentLoader', this.presentLoader);
     emitter.on('dismissLoader', this.dismissLoader);
+    updateToken(this.userToken)
```

Comment:
We are setting these values in main.ts as well
https://github.com/hotwax/import/pull/47/files#diff-4fab5baaca5c14d2de62d8d2fceef376ddddcc8e9509d86cfa5643f51b89ce3dR67

This is an redundant initialisation, we could make it at single place

Reply:
Removed the initialization from the main file and moved it to `App.vue`.